docs/config/node.rst

```diff
@@ -205,6 +208,65 @@ Ports
   range is used. If this is set to an integer value it is considered as an
   explicit single port.
 
+Advanced TCP settings
+---------------------
+
+Any interface that uses TCP (PG/HTTP/transport) shares the following settings:
+
+.. _network.tcp.no_delay:
+
+**network.tcp.no_delay**
+  | *Default:* ``true``
+  | *Runtime:* ``no``
+
+  Enable or disable the `Nagle's algorithm`_ for buffering TCP packets.
+  Buffering is disabled by default.
+
+.. _network.tcp.keep_alive:
+
+**network.tcp.keep_alive**
+  | *Default:* ``true``
+  | *Runtime:* ``no``
+
+  Configures the ``SO_KEEPALIVE`` option for sockets, which determines
+  whether they send TCP keepalive probes.
+
+.. _network.tcp.reuse_address:
+
+**network.tcp.reuse_address**
+  | *Default:* ``true on non-windows machines and false otherwise``
+  | *Runtime:* ``no``
+
+   Configures the `` SO_REUSEADDRS`` option for sockets, which determines
+   whether they should reuse the address.
+
+.. _network.tcp.send_buffer_size:
+
+**network.tcp.send_buffer_size**
+  | *Default:* ``-1``
+  | *Runtime:* ``no``
+
+  The size of the TCP send buffer. By default not explicitly set.
+
+.. _network.tcp.receive_buffer_size:
+
+**network.tcp.receive_buffer_size**
+  | *Default:* ``-1``
+  | *Runtime:* ``no``
+
+  The size of the TCP receive buffer. By default not explicitly set.
+
+.. NOTE::
+
+    Each setting in this section has its counterpart for HTTP and transport.
+    To provide a protocol specific setting, remove ``network`` prefix and use
+    either ``http`` or ``transport`` instead. For example, no_delay can be
+    configured as ``http.tcp.no_delay`` and ``transport.tcp.no_delay``. Please
+    note, that PG interface takes its settings from transport.
```

Comment:
For HTTP we have `Netty4HttpServerTransport`. Last sentence that PG settings = transport settings is written according to `PgClient` configuration code.